packages/server/src/authentication/authentication.service.ts

```diff
@@ -69,21 +73,21 @@ export default class AuthenticationService {
       throw new EmailVerificationTokenExpired();
     }
 
-    const user = await this.userRepository.findOne({ id: emailVerification.userId });
+    const user = await this.userRepository.findOne({
+      id: emailVerification.userId,
+    });
     user.status = UserStatuses.ACTIVE;
     await this.userRepository.update(user.id, user);
     await this.emailVerificationRepository.delete(emailVerification.id);
 
     return this.loginUser(user);
   }
 
-  private loginUser(user: User): UserWithToken {
+  private loginUser(user: UserEntity): UserWithToken {
     const tokenData = createToken(user);
 
     return {
-      id: user.id,
-      name: user.name,
-      email: user.email,
+      ...user,
```

Comment:
I don't think that we need to send all the user data including password.

Reply:
I believe this is the user info from our DB is it not? Therefore it should not contain any password. Or does it?

Reply:
It contains a hashed password, but I don't think that we should send not needed and still sensitive data to clients.

Reply:
Well, accroding to the interface we do return user with token here, do we not?

Reply:
@karolinayanko agree, but that is rather a tech debt than a part of this ticket, what do you think?